### PR TITLE
replace docker references with acr dev repo and stop publishing to docker

### PR DESCRIPTION
**Pull request by arroyc**

### CODE REVIEW

File: build/buildBuildImages.sh [hunk 1]
```diff
@@ -116,18 +114,14 @@ then
...
-	echo
-	echo "List of images built (from '$BUILD_IMAGES_ARTIFACTS_FILE'):"
-	cat $BUILD_IMAGES_ARTIFACTS_FILE
-	echo "List of images tagged (from '$ACR_BUILD_IMAGES_ARTIFACTS_FILE'):"
-	cat $ACR_BUILD_IMAGES_ARTIFACTS_FILE
 fi
 
+echo
+echo "List of images tagged (from '$ACR_BUILD_IMAGES_ARTIFACTS_FILE'):"
+cat $ACR_BUILD_IMAGES_ARTIFACTS_FILE
```
Comment (dorfire) [inline build/buildBuildImages.sh]: 👍

File: build/buildRunTimeImages.sh [hunk 2]
```diff
@@ -64,6 +64,9 @@ fi
 for generateDockerFile in $generateDockerFiles; do
     echo
     echo "Executing '$generateDockerFile'..."
+	ls -l "$generateDockerFile"
```
Comment (dorfire) [inline build/buildRunTimeImages.sh]: why twice?

File: build/buildRunTimeImages.sh [hunk 3]
```diff
@@ -64,6 +64,9 @@ fi
...
+	chmod +x "$generateDockerFile"
+	ls -l "$generateDockerFile"
```
Comment (dorfire) [inline build/buildRunTimeImages.sh]: indentation

File: images/runtime/node/generateDockerfiles.sh [hunk 4]
```diff
@@ -10,7 +10,7 @@ declare -r DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 declare -r DOCKERFILE_TEMPLATE="$DIR/Dockerfile.template"
```
Comment (dorfire) [inline images/runtime/node/generateDockerfiles.sh]: image

File: build/buildRunTimeImages.sh (hunk 2 above)
Comment (arroyc) [reply build/buildRunTimeImages.sh]: ah this needs to be removed … this was for that weird permission issue

File: images/runtime/node/generateDockerfiles.sh (hunk 4 above)
Comment (arroyc) [reply images/runtime/node/generateDockerfiles.sh]: not sure how this happened

File: build/buildBuildImages.sh [hunk 5]
```diff
@@ -48,7 +48,7 @@ fi
 function BuildAndTagStage()
```
Comment (dorfire) [inline build/buildBuildImages.sh]: the permissions of this file were changed from 755 -> 644

File: build/buildRunTimeImages.sh [hunk 6]
```diff
@@ -64,6 +64,7 @@ fi
 for generateDockerFile in $generateDockerFiles; do
     echo
     echo "Executing '$generateDockerFile'..."
+    chmod +x "$generateDockerFile"
```
Comment (dorfire) [inline build/buildRunTimeImages.sh]: is still this needed, even after that `generateDockerfiles.sh` was given the correct permissions?

Comment (arroyc) [reply build/buildRunTimeImages.sh]: I kept it for safety, in case it happens again it's hard to find compared to other scripts, what do you think?

**Pull request merged**